### ISSUE TITLE
QOL:Camera Distance 1200>1350-1450

### ISSUE THREAD

**Issue by Avarise831** [ValveSoftware/Dota2-Gameplay#1281]
### Description

/steam/steamapps/common/dota2beta/game/dota/bin/win64/client/dll 
open with notepad++,CTRL+F type  1200 (found green 1200) and change on 1350 or 1450 
i mean  legal or not?can we have add this options in game client?because after each update you need to change default value
this one also need to add in game client options
dota_selection_groups false/true  
dota_minimap_creep_scale 1,2,3

**Comment by PikaChokeMe** [comment]
> /steam/steamapps/common/dota2beta/game/dota/bin/win64/client/dll
> open with notepad++,CTRL+F type 1200 (found green 1200) and change on 1350 or 1450

You are _modifying_ _**game binaries**_ for this. I think almost everyone would consider this _**cheating.**_

**Comment by Avarise831** [comment]
> > /steam/steamapps/common/dota2beta/game/dota/bin/win64/client/dll
> > open with notepad++,CTRL+F type 1200 (found green 1200) and change on 1350 or 1450
> 
> `client.dll` is a game _binary._ You are _modifying_ _**game binaries**_ for this. I think almost everyone would consider this _**cheating.**_

so many people have been using this for many years i just wish i could make it available to everyone.i think its like autoexec.cfg

**Comment by shaileshgarg** [comment]
This is how some cheats work. Editing DLL files can lead to a vac ban

**Comment by Avarise831** [comment]
> This is how some cheats work. Editing DLL files can lead to a vac ban

yes, but there has never been a single case of a ban for increase camera distance, although a lot of people like me have been using it for 6+ years, so it needs to be added to the game client or banned
